### PR TITLE
Allow Doctrine Event manager v2

### PR DESCRIPTION
**Pull request by X-Coder264**
## Subject

I am targeting this branch, because this is a backwards compatible change.

## Changelog

```markdown
### Added
- Support for Doctrine Event manager v2
```

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
         "doctrine/collections": "^1.6",
         "doctrine/common": "^3.1",
         "doctrine/dbal": "^2.13 || ^3.0",
-        "doctrine/event-manager": "^1.1",
+        "doctrine/event-manager": "^1.2 || ^2.0",
```
Comment (X-Coder264) [inline composer.json:16]: The 1.1 to 1.2 bump is needed in order to be able to have code that is compatible with both v1 and v2 at the same time (the `getAllListeners` method call to be more specific).

**Review by VincentLanglet — APPROVED** [review]

**Comment by VincentLanglet** [comment]
Thanks !

Do you need a release for this ?

**Comment by X-Coder264** [comment]
@VincentLanglet Thanks for the merge. I'm in no hurry for a release for this, though https://github.com/sonata-project/EntityAuditBundle/pull/543 probably warrants a release once it's merged.

**Comment by VincentLanglet** [comment]
#543 is merged too. I can make a release, unless you have other PR in mind before

**Comment by X-Coder264** [comment]
@VincentLanglet I have one more PR in mind, will be sending it in today.

**Comment by X-Coder264** [comment]
@VincentLanglet Here it is, even sooner than I expected :smile: 

https://github.com/sonata-project/EntityAuditBundle/pull/544

**Pull request merged**